lua/luasnip/extras/init.lua

```diff
@@ -150,8 +151,14 @@ return {
 			})
 		end, args_indcs)
 	end,
+	format_nodes = format_nodes,
 
 	--alias
 	l = lambda,
 	m = match,
+	fmt = format_nodes,
+	fmta = function(str, nodes, opts)  -- fmt with angle brackets
+		opts = vim.tbl_extend('force', opts or {}, { delimiters = '<>' })
+		return format_nodes(str, nodes, opts)
+	end,
```

Comment:
Actually, could you leave it as just a submodule for `extras`? (just sticking with `luasnip.extras.fmt` is alright imo) (I'm eventually going to split `luasnip.extras` up a bit, will be one less breaking change then :D)

Reply:
Done